py/desispec/qa/qalib.py

```diff
@@ -301,8 +301,7 @@ def SN_ratio(flux,ivar):
     medsnr=np.zeros(flux.shape[0])
     #totsnr=np.zeros(flux.shape[0])
     for ii in range(flux.shape[0]):
-        signalmask=flux[ii,:]>0 #- mask negative values
-        snr=flux[ii,signalmask]*np.sqrt(ivar[ii,signalmask])
+        snr=flux[ii]*np.sqrt(ivar[ii])
         medsnr[ii]=np.median(snr)
```

Comment:
@sbailey We decided to change this to reduce a biased S/N calculation, but input would be appreciated. The plots below show the SNR QA with all values (top) and no negative values (bottom), if that helps.

![allvalues](https://cloud.githubusercontent.com/assets/15023569/23391583/be4d1630-fd3b-11e6-8b6e-89425740ce28.png)
![no_negatives](https://cloud.githubusercontent.com/assets/15023569/23391593/c9a73e16-fd3b-11e6-9ed5-5edd34900ca8.png)